perfkitbenchmarker/providers/gcp/util.py

```diff
@@ -26,7 +26,7 @@
 @functools32.lru_cache()
 def GetDefaultProject():
   """Get the default project."""
-  cmd = ['gcloud', 'config', 'list', '--format=json']
+  cmd = [FLAGS.gcloud_path, 'config', 'list', '--format=json']
```

Comment:
It is from https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/commit/3ff5327958c0b16c38b7685927f35b6767cb20a3

You just need to rebase on master I believe.